src/Perfmon.h

```diff
@@ -0,0 +1,163 @@
+/**@file
+ * This file is part of the TASTE Leon3 Runtime.
+ *
+ * @copyright 2022 N7 Space Sp. z o.o.
+ *
+ * TASTE Leon3 Runtime was developed under a programme of,
```

Comment:
Please change this to GPL and remove the "developed... and funded by". This will need to be updated properly in the near future.

Reply:
Done

Reply:
![image](https://user-images.githubusercontent.com/17752977/194545670-e9150ba0-34e9-4614-9b32-5f20eaef5f60.png)
Is it though?